src/components/Toast/types.tsx

```diff
@@ -29,3 +30,8 @@ export const toastStylesBackgroundColorsMap: Record<ToastStyle, string> = {
     [ToastStyle.Success]: 'tw-bg-box-positive-strong-pressed',
     [ToastStyle.Warning]: 'tw-bg-box-warning-strong',
 };
+
+export enum BorderStyle {
+    Default = 'Default',
+    Mighty = 'Mighty',
+}
```

Comment:
Since this is specific to `Toast` maybe we should use a `map` `Record` like we do for the background?  Then implement is directly in the component?